The shutdown binary is actually provided by runit and expects the runit
init to be running. This is not the case in the initramfs. Forcing the
shutdown, halt or reboot is necessary to ignore this requirement.

---
 modules.d/99base/dracut-lib.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -1004,13 +1004,13 @@ emergency_shell() {
 
     case "$_emergency_action" in
         reboot)
-            reboot || exit 1
+            reboot -f || exit 1
             ;;
         poweroff)
-            poweroff || exit 1
+            poweroff -f || exit 1
             ;;
         halt)
-            halt || exit 1
+            halt -f || exit 1
             ;;
     esac
 }
-- 
2.37.3

